.travis.yml

```diff
@@ -6,16 +6,20 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
-  - ruby-head
-  - jruby-head
+  # - ruby-head
+  # - jruby-head
```

Comment:
need that back ?

Reply:
Oh yeah, I commented those out because they are consistently failing and adding 23 minutes to the build times, because Travis CI runs all of the builds in a sequence (not in parallel.)

See:

* https://travis-ci.org/grosser/parallel_tests/builds/621971355?utm_medium=notification&utm_source=github_status
* https://travis-ci.org/grosser/parallel_tests/builds/621971005?utm_medium=notification&utm_source=github_status
* https://travis-ci.org/grosser/parallel_tests/builds/621957792?utm_medium=notification&utm_source=github_status

I had a quick look to see if I could fix those builds as well, but it was a bit difficult. The `ruby-head` one was particularly weird, because [it's just getting stuck at the installation/gemset stage with RVM](https://travis-ci.org/grosser/parallel_tests/jobs/621971011?utm_medium=notification&utm_source=github_status) and timing out after 10m of inactivity. I found these related issues but couldn't find a solution:

* https://github.com/rvm/rvm/issues/3344
  * Something to do with DNS timeouts?
* https://github.com/rvm/rvm/issues/2455

But I will uncomment these!